contracts/stellar-axelar-gateway/src/storage.rs

```diff
@@ -2,22 +2,21 @@ use soroban_sdk::{contracttype, BytesN, String};
 use stellar_axelar_std::contractstorage;
 
 #[contracttype]
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub struct MessageApprovalKey {
     pub source_chain: String,
     pub message_id: String,
 }
 
 #[contracttype]
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(PartialEq)]
```

Comment:
clippy warns that `Eq` should be derived as if `PartialEq` is provided. Good practice in general, but it's not really needed for storage data key and error types. We could whitelist it in Cargo.toml